src/main/java/net/kodehawa/mantarobot/core/command/TextContext.java

```diff
@@ -487,6 +527,22 @@ public Player getPlayer(String id) {
         return managedDatabase.getPlayer(id);
     }
 
+    public PlayerStats getPlayerStats() {
+        return managedDatabase.getPlayerStats(getMember());
+    }
+
+    public PlayerStats getPlayerStats(String id) {
+        return managedDatabase.getPlayerStats(id);
+    }
+
+    public PlayerStats getPlayerStats(User user) {
+        return managedDatabase.getPlayerStats(user);
+    }
+
+    public PlayerStats getPlayerStats(Member member) {
+        return managedDatabase.getPlayerStats(member);
+    }
+
```

Comment:
Both TextContext and BaseInteractionContext now have these methods and both implementations have the exact same bodies. These should really be a default methods in IContext.